docs_espressif/en/settings.rst

```diff
@@ -1,258 +1,289 @@
+
+.. list-table::
+    :widths: 25 75
+    :header-rows: 1
+
+    * - Setting
+      - Description
+    * - **idf.flashBaudRate**
+      - Flash Baud Rate
+    * - **idf.monitorBaudRate**
+      - Monitor Baud Rate (Empty by default to use SDKConfig ``CONFIG_ESP_CONSOLE_UART_BAUDRATE``)
+    * - **idf.openOcdConfigs**
+      - Configuration files for OpenOCD, relative to ``OPENOCD_SCRIPTS`` folder
+    * - **idf.openOcdLaunchArgs**
+      - Launch arguments for OpenOCD before ``idf.openOcdDebugLevel`` and ``idf.openOcdConfigs``
+    * - **idf.openOcdDebugLevel**
+      - Set OpenOCD Debug Level (0-4) Default: 2
+    * - **idf.port**
+      - Path of selected device port
+    * - **idf.portWin**
+      - Path of selected device port in Windows
+    * - **idf.enableSerialPortChipIdRequest**
+      - Enable detecting the chip ID and show on serial port selection list
+    * - **idf.useSerialPortVendorProductFilter**
+      - Enable use of ``idf.usbSerialPortFilters`` list to filter serial port devices list
+    * - **idf.usbSerialPortFilters**
+      - USB productID and vendorID list to filter known Espressif devices
+    * - **openocd.jtag.command.force_unix_path_separator**
+      - Forced to use ``/`` instead of ``\\`` as path separator for Win32 based OS
+    * - **idf.svdFilePath**
+      - SVD file absolute path to resolve chip debug peripheral tree view
 
 This is how the extension uses them:
 
-1. **idf.flashBaudRate** is the baud rate value used for the **ESP-IDF: Flash your Project** command and [ESP-IDF Debug](./DEBUGGING.md).
-2. **idf.monitorBaudRate** is the ESP-IDF Monitor baud rate value and fallback from your project's skdconfig ``CONFIG_ESPTOOLPY_MONITOR_BAUD`` (idf.py monitor' baud rate). This value can also be override by setting the environment variable ``IDF_MONITOR_BAUD`` or ``MONITORBAUD`` in your system environment variables or this extension's **idf.customExtraVars** configuration setting.
-3. **idf.openOcdConfigs** is used to store an string array of OpenOCD scripts directory relative path config files to use with OpenOCD server. (Example: ``["interface/ftdi/esp32_devkitj_v1.cfg", "board/esp32-wrover.cfg"]``). More information `OpenOCD JTAG Target configuration <https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/jtag-debugging/tips-and-quirks.html#jtag-debugging-tip-openocd-configure-target>`_.
+1. **idf.flashBaudRate** is the baud rate value used for the **ESP-IDF: Flash your Project** command and `Debugging <https://github.com/espressif/vscode-esp-idf-extension/blob/master/docs/DEBUGGING.md>`_.
+2. **idf.monitorBaudRate** is the ESP-IDF Monitor baud rate value and fallback from your project's sdkconfig ``CONFIG_ESPTOOLPY_MONITOR_BAUD`` (idf.py monitor' baud rate). This value can also be override by setting the environment variable ``IDF_MONITOR_BAUD`` or ``MONITORBAUD`` in your system environment variables or this extension's **idf.customExtraVars** configuration setting.
```

Comment:
```suggestion
2. **idf.monitorBaudRate** is the ESP-IDF Monitor baud rate value and fallback from your project's sdkconfig ``CONFIG_ESPTOOLPY_MONITOR_BAUD`` (idf.py monitor' baud rate). This value can also be overridden by setting the environment variable ``IDF_MONITOR_BAUD`` or ``MONITORBAUD`` in your system environment variables or in the **idf.customExtraVars** setting of the extension.
```

Reply:
Hi @0cici, 
I think "in the idf.customExtraVars setting of the extension" might cause confusion, as it appears parallel to "in your system environment variables". To improve clarity, I prefer adjusting the syntax and rewriting the sentence in the active voice. Pls check the re-edited sentence. Thx!